### DISCUSSION TITLE
looking for a remote unpaid internship

### DISCUSSION THREAD

**Discussion by atrgouti** [community/community#131078] [Programming Help]
### Select Topic Area

Question

### Body

Hi, I just graduated and I am looking for an internship. I did not think that it would be this hard to find an internship as a developer. I have good skills, especially in frontend development. I am intermediate to expert in React. I don't care about the money or how much I make; all I want is to work on interesting projects and learn new things that are practical in the real world. Here is my portfolio: [www.atrgouti.me](http://www.atrgouti.me/). If anyone can help or has some tips, I would really, really appreciate it. I didn't come here to ask for help until I tried all other ways but failed. Thank you for your attention.

**Comment by ebndev** [comment]
Hi @atrgouti, thanks for sharing your portfolio with the community!

Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Please feel encouraged to post in the future if you have questions and best of luck on your journey!
